change-validation/src/main/scala/com/normation/plugins/changevalidation/NotificationService.scala

```diff
@@ -246,7 +264,7 @@ class NotificationService(
       , "description" -> cr.info.description
       , "author"      -> cr.owner
       // rudderBaseUrl should not contains "/" at the end
-      , "link"        -> (rudderBaseUrl + RudderConfig.linkUtil.changeRequestLink(cr.id))
+      , "link"        -> (rudderBaseUrl + linkUtil.baseChangeRequestLink(cr.id))
```

Comment:
`baseChangeRequestLink` is necessary: `changeRequestLink` uses lift `S` (ie request session) to compute the base path - what we wanted to avoid by asking the user for the `rudderBaseUrl` to use.